dev/CommandBarFlyout/CommandBarFlyout.cpp

```diff
@@ -252,26 +252,135 @@ winrt::Control CommandBarFlyout::CreatePresenter()
+        // closed. In order for drop shadows to render correctly, the element with the shadow needs to have
+        // the correct corner radius as well. Otherwise the shadow will render with sharp corners.
+        if (SharedHelpers::Is21H1OrHigher())
+        {
+            presenterControl7.CornerRadius(primaryCornerRadius);
+        }
+        else
         {
-            presenter2.IsDefaultShadowEnabled(false);
+            presenterControl7.CornerRadius({ 0 });
+        }
+    }
+
+    m_presenter.set(presenter);
+
+    m_commandBarOpenedRevoker = commandBar->Opened(winrt::auto_revoke, {
+        [this, primaryCornerRadius](auto const&, auto const&)
+        {
+            if (winrt::IFlyoutBase5 thisAsFlyoutBase5 = *this)
+            {
+                // If we open the CommandBar, then we should no longer be in a transient show mode -
+                // we now know that the user wants to interact with us.
+                thisAsFlyoutBase5.ShowMode(winrt::FlyoutShowMode::Standard);
+            }
+
+            if (SharedHelpers::Is21H1OrHigher() && m_presenter)
+            {
+                if (winrt::IControl7 presenterControl7 = m_presenter.get())
+                {
+                    if (auto commandBar = winrt::get_self<CommandBarFlyoutCommandBar>(m_commandBar.get()))
+                    {
+                        if (m_secondaryCommands.Size() > 0 || commandBar->IsOpen())
+                        {
+                            auto cornerRadiusConverter = winrt::make_self<CornerRadiusFilterConverter>();
+
+                            // CommandBarFlyout already changes its corner radius depending on if the overflow is open
+                            // and in which direction through visual states. Since we're managing the corner radius
+                            // on the presenter manually, we'll need to mimic these changes onto the presenter's corner radius.
+                            if (commandBar->IsExpandedUp())
+                            {
+                                presenterControl7.CornerRadius(cornerRadiusConverter->Convert(primaryCornerRadius, winrt::CornerRadiusFilterKind::Bottom));
+                            }
+                            else
+                            {
+                                presenterControl7.CornerRadius(cornerRadiusConverter->Convert(primaryCornerRadius, winrt::CornerRadiusFilterKind::Top));
+                            }
+                        }
+                    }
+                }
+            }
         }
+    });
+
+    if (SharedHelpers::Is21H1OrHigher())
+    {
+        // Since DropShadows don't play well with the entrance animation for the presenter,
+        // we'll need to fade it in. This name helps us locate the element to set the flag.
+        // We didn't want to apply this name to all FlyoutPresenters, which is why it's done
+        // in code behind here.
+        presenter.Name(L"DropShadowFadeInTarget");
```

Comment:
>DropShadowFadeInTarget [](start = 25, length = 22)

I don't see this name being used anywhere, what am I missing?

Reply:
It's used in OS code when setting the fade in flag for a drop shadow.